release notes/v0.36.0.md

```diff
@@ -0,0 +1,129 @@
+k6 v0.36.0 is here! 🎉 It introduces a couple of new features which enhance its usability, includes a number of fixes, and the result of ongoing refactoring efforts.
+
+In total, we have closed 27 issues.
```

Comment:
```suggestion
```

Not sure if this is accurate, or if it is, it's a very subjective number. A lot of the things we changed this cycle were internal refactoring, some of which didn't have an issue or a bunch were under the same issue. Since it's subjective and doesn't really inform users of anything, I'm not sure if should be even mentioned :thinking:

Reply:
Makes sense, I simply updated the number that was present in `v0.35.0`. Will remove it indeed 👍🏻